src/header/Header.scss

```diff
@@ -1,5 +1,56 @@
+@import "../utils/Mixins";
 @import "../utils/Variables";
 
+@include media-breakpoint-down(sm) {
+  header {
+    flex-wrap: wrap;
+
+    & nav,
+    nav ul,
+    nav ul li {
+      width: 100%;
+    }
+
+    & nav ul {
+      margin-top: 24px;
+    }
+
+    & .navLink {
```

Comment:
I guess it does not need to be in header to work? so let's move it up just inside @media.